package.json

```diff
@@ -2,7 +2,6 @@
   "name": "scigateway",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://localhost:8000/",
```

Comment:
To avoid CORS issues when booting up the server for development, I would suggest keeping the proxy config in `package.json` and setting the default auth sever URL in `settings.example.json` to `http://localhost:3000` - this makes things work with CORS in the default set up but still makes it clear that a URL is expected for the value of that setting.